README.md

```diff
@@ -1,4 +1,4 @@
-![Gradle Checks](https://github.com/hhu-propra2/abschlussprojekt-mopse/workflows/Gradle%20Checks/badge.svg)
+![Gradle Checks](https://github.com/hhu-propra2/abschlussprojekt-mopse/workflows/Gradle%20Checks/badge.svg) ![Generate Documentation](https://github.com/hhu-propra2/abschlussprojekt-mopse/workflows/Generate%20Documentation/badge.svg)
```

Comment:
does the badge link work as you changed the procedure multiple times

Reply:
Yes I think so, but we could use as link to the documentation?

Reply:
the badge is fine, i just want to know if the link is still valid?
can you regenerate the badge link to be sure?

Reply:
sure thing

Reply:
regenerated it, same link as befor